libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/view/MapboxAudioGuidanceButton.kt

```diff
@@ -0,0 +1,224 @@
+package com.mapbox.navigation.dropin.view
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.annotation.DrawableRes
+import androidx.annotation.StyleRes
+import androidx.annotation.UiThread
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.databinding.MapboxAudioGuidanceButtonLayoutBinding
+import com.mapbox.navigation.ui.utils.internal.ExtendableButtonHelper
+import com.mapbox.navigation.ui.utils.internal.extensions.measureTextWidth
+
+/**
+ * Default button that allows user to mute and un-mute audio guidance.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class MapboxAudioGuidanceButton : FrameLayout {
```

Comment:
Curious about this, it can't be an ExtendableButton and will it replace MapboxSoundButton?

Reply:
> it can't be an ExtendableButton
Is this a question?

>will it replace MapboxSoundButton
Yes that's the idea eventually

Reply:
> > it can't be an ExtendableButton

> Is this a question?

Yeah just a question, thought we were going to use `ExtendableButton`. But am now noticing `MapboxCameraModeButton` is also a custom button - appears very similar to MapboxAudioGuidanceButton 🤔